types/aws-lambda/trigger/lex-v2.d.ts

```diff
@@ -24,7 +29,7 @@ export interface LexV2Bot {
     aliasId: string;
     aliasName: string;
     localeId: string;
-    version: string;
+    version: 'DRAFT' | `${number}`;
```

Comment:
I think this is probably overkill? Are you likely to care about the specific format, as opposed to just logging it?

Reply:
Removed in latest commit.